collector/collector/report_handler.py

```diff
@@ -41,7 +44,8 @@
 
 # see config.py for the meaning of this value
 TELEMETRY_ID = app.config.get("TELEMETRY_ID", "6907c830-eed9-4ce9-81ae-76daf8d88f0f")
-
+ATTACHMENT_QUARANTINE_FOLDER = app.config.get("ATTACHMENT_QUARANTINE_FOLDER")
```

Comment:
Might want a builtin default value in case the config does not define `ATTACHMENT_QUARANTINE_FOLDER`.

Reply:
Yes, good practice and consistent with other configurations.